src/stats.bif

```diff
@@ -93,7 +88,8 @@ function get_conn_stats%(%): ConnStats
 
 	r->Assign(n++, val_mgr->GetCount(Connection::TotalConnections()));
 	r->Assign(n++, val_mgr->GetCount(Connection::CurrentConnections()));
-	r->Assign(n++, val_mgr->GetCount(Connection::CurrentExternalConnections()));
+        // TODO: is it ok to remove the field from the object? Does that break anything else?
+	r->Assign(n++, val_mgr->GetCount(0));
```

Comment:
Doubt it's used much, but yeah safe thing I think is set it to 0 here and then also can add `&deprecated` to where that field is defined in the `ConnStats` record in `init-bare.zeek`.

Reply:
Or as I notice you went ahead and removed `is_external_connection` and I think about it a bit more, maybe ok to go ahead and remove that field instead of deprecate since deprecation is more about giving people a grace period where the functionality is still working, but here things already don't work and they'll just need to remove any usages in their scripts, which I hope is not too complicated of an ask.  But do call out the removals in NEWS.

Reply:
A little surprisingly, we have no btests that fail when I remove that field. That's a little concerning.